website/docs/d/notification_destination.html.markdown

```diff
@@ -30,6 +30,10 @@ resource "newrelic_notification_channel" "foo-channel" {
     value = "{\n\t\"name\": \"foo\"\n}"
     label = "Payload Template"
   }
+
+  secure_url {
+    prefix = "https://webhook.site/"
+  }
```

Comment:
@nzur-newrelic I think this is incorrect, since the `newrelic_notification_channel` resource does not support an argument `secure_url` and this change adds the argument to the resource. Am I missing something here?

Reply:
Same